### PR TITLE
Fix #13858: Removed velocity settings for dynamics and hairpins

### LINKED ISSUE #13858: Remove velocity settings for dynamics and hairpins
**Task description**
This is a step we need to take to have a simpler, universal system for velocity change in MuseScore 4. In MuseScore 3, there are competing velocity parameters which the user then needs to be able to learn to either combine or prioritise. It is too complex and unworkable for the future.

So for MS4, velocity can only be placed on notes themselves.

Hence, these options need to be removed of hairpins and dynamics:

image

image

---

Moreover, we now should also remove the tab in 'playback' that distinguishes between 'General' and 'Dynamics'
image

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #13858

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Tantacrul** [comment]
This MacOS check is taking forever :)

**Comment by Eism** [comment]
restarted

**Comment by Tantacrul** [comment]
OK, just tested. Looking good!

@Eism or @RomanPudashkin - if you're happy, can this baby be floated down the river merge?

File: src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackDynamicsSettings.qml [hunk 1]
```diff
@@ -55,13 +55,6 @@ Item {
         }
 
         SeparatorLine { anchors.margins: -12 }
```
Comment (Eism) [inline src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackDynamicsSettings.qml:57]: line should also be removed

Comment (HemantAntony) [reply src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackDynamicsSettings.qml]: I have updated the PR again :)

**Pull request merged**